pkg/bloomgateway/processor.go

```diff
@@ -145,6 +145,11 @@ func (p *processor) processBlock(_ context.Context, bq *bloomshipper.CloseableBl
 		return err
 	}
 
+	// We require V3 schema
+	if !schema.IsCurrentSchema() {
+		return v1.ErrUnsupportedSchemaVersion
```

Comment:
I noticed that `processor.processTasks` aborts after the first task group where `p.processTasksForDay` returns an error. Is that something we want to keep?

Especially with this most recent change, I believe that means if any of the existing blocks aren't the current schema version, then we don't process any of newer ones, even if they're valid.